File: C/Chapter 3 - Control Statement/2K221008/patteren_star_descending.c
//Loops:

//Pattern Programs

/*

Ex. Star Pattern - Descending

* * * * *
* * * *
* * *
* *
*

*/

#include<stdio.h>
void main()
{
    int count,i,j;

    printf("Enter the count: ");
    scanf("%d",&count);
    printf("\n");

    for(i=1; i <= count; i++){
        for (j=i;j<=count;j++){
            printf("* ");
        }
        printf("\n");
    }

}

File: Back End/Python/Chapter 05 - Control Statements/2K221229/patteren_star_descending.py
# //Loops:
#
# //Pattern Programs
#
# /*
#
# Ex. Star Pattern - Descending
#
# * * * * *
# * * * *
# * * *
# * *
# *
#
# */

limit = int(input("Enter the limit: "))

for i in range(0, limit, 1):
    for j in range(i, limit, 1):
        print("* ", end=' ')
    print(end='\n')
